pkg/autodiscovery/integration/config.go

```diff
@@ -270,9 +270,7 @@ func (c *Config) Digest() string {
 	for _, i := range c.ADIdentifiers {
 		h.Write([]byte(i))
 	}
-	for _, i := range c.EndpointsChecks {
-		h.Write([]byte(i.Digest()))
-	}
+	h.Write([]byte(c.NodeName))
```

Comment:
No, because the entity in each config is unique (as we use IPs to differentiate)
We hash the `ADIdentifiers` and by default we add the `Entity` as `ADIdentifiers` every time we generate a config
```
for _, i := range c.ADIdentifiers {
		h.Write([]byte(i))
	}
	h.Write([]byte(c.NodeName))
```


```
// GetADIdentifiers returns the service AD identifiers
func (s *KubeEndpointService) GetADIdentifiers() ([]string, error) {
	return []string{s.entity}, nil
}
```

Reply:
Ah right, I missed that then, all good indeed